packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -189,6 +190,18 @@ export const Table = memo((props: Props) => {
 
   const pageSize = Math.round(listHeight / tableStyles.rowHeight) - 1;
 
+  // Make sure we have room to show the sub-table
+  const expandedIndices = Object.keys(extendedState.expanded);
+  if (expandedIndices.length) {
+    const subTablesHeight = expandedIndices.reduce((sum, index) => {
+      const subLength = subData?.find((frame) => frame.meta?.custom?.parentRowIndex === parseInt(index, 10))?.length;
```

Comment:
I'm a bit surprised this isn't an integer comparison. Am I missing something?

Reply:
Hey Andre, yes that's correct. Didn't realize the internal react table state used a string index so no worries there!